source/Mlos.Agent.Server/MlosAgentServer.cs

```diff
@@ -75,15 +87,24 @@ public static void Main(string[] args)
                 //
                 AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
 
+                // This populates a variable for the various settings registry
+                // callback handlers to use (by means of their individual
+                // AssemblyInitializers) to know how they can connect with the
+                // optimizer.
+                //
+                // See Also: AssemblyInitializer.cs within the SettingsRegistry
+                // assembly project in question.
+                //
                 Uri optimizerAddressUri = new Uri("http://localhost:50051");
                 MlosContext.OptimizerFactory = new MlosOptimizer.BayesianOptimizerFactory(optimizerAddressUri);
             }
 
-            // Create circular buffer shared memory before running the target process.
+            // Create (or open) the circular buffer shared memory before running the target process.
             //
             MainAgent.InitializeSharedChannel();
 
-            // Active learning, almost done. In active learning the MlosAgentServer controls the workload against the target component.
+            // Active learning mode.
+            // In active learning the MlosAgentServer controls the workload against the target component.

```

Comment:
I don't see how/where the workload is controlled. It just launches an executable, right?

Reply:
You're right, I think that is probably still future/planned work.  I'll tag it as such.